[2021-10-02 04:10:37,573] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_C 2021-10-01T00:00:00+00:00 [queued]>
[2021-10-02 04:10:37,745] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_C 2021-10-01T00:00:00+00:00 [queued]>
[2021-10-02 04:10:37,746] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 04:10:37,748] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-02 04:10:37,750] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 04:10:37,818] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_C> on 2021-10-01T00:00:00+00:00
[2021-10-02 04:10:37,883] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_C', '2021-10-01T00:00:00+00:00', '--job-id', '10', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp3nt2eksg', '--error-file', '/tmp/tmp0i0zpdx_']
[2021-10-02 04:10:37,903] {standard_task_runner.py:77} INFO - Job 10: Subtask Training_Model_C
[2021-10-02 04:10:37,842] {standard_task_runner.py:52} INFO - Started process 17335 to run task
[2021-10-02 04:10:38,173] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_C 2021-10-01T00:00:00+00:00 [running]> on host d6b4131456ad
[2021-10-02 04:10:38,917] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_C
AIRFLOW_CTX_EXECUTION_DATE=2021-10-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-01T00:00:00+00:00
[2021-10-02 04:10:38,929] {python.py:151} INFO - Done. Returned value was: 6
[2021-10-02 04:10:39,122] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_C, execution_date=20211001T000000, start_date=20211002T041037, end_date=20211002T041039
[2021-10-02 04:10:39,251] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-02 04:10:39,497] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
